### PR TITLE
Onboarding fixups

### PR DESCRIPTION
**Pull request by nathanleiby**
Thanks for the excellent setup guide. I just got the repo running locally and connected to a Google Developer account. 

This PR has a few suggestions to clarify the "Getting Started" steps in the README.  (1808bac, 5544bd6)
It also suggests including a `.nvmrc` file, to encode the node version for developers. (634600c)

See commit details for more info.

### CODE REVIEW

File: README.md
Comment (nathanleiby) [inline README.md:34]: I think our goal here is

1. be able to create a spreadsheet (`drive.file`) a
2. be able to read from that spreadsheet (`auth/spreadsheets.readonly` might be sufficient -- so far I used read/write permissions)

https://developers.google.com/sheets/api/guides/authorizing

**Comment by nathanleiby** [comment]
The build failure is due to failing tests. Seems unrelated to my change, as I haven't modified any code. 

Running `npm test` locally, I see similar failures (a subset of those seen in CI):
2__zsh

Failures seen in CI:
Onboarding_fixups_by_nathanleiby_·_Pull_Request__102_·_paperize_paperize

File: README.md
Comment (lorennorman) [reply README.md:34]: I'll be honest: the Google permissions are confusing to get right.

What we settled on is that we only want permissions that need no special approval from Google. `drive.file` fits this, but I still haven't figured out exactly how it works. Basically, you have full read/write access to any files that you create, but if the user drops a file you DIDN'T create into the Paperize folder, Paperize can't see it. That seems wrong, seems like there should be a way to flag that file as available to Paperize and enable that use case (instead of the janky file uploader I had to write into Paperize itself.)

I spent way too many hours trying to get different things to work, so I'm not looking to spend anymore as what we have works and is quite secure (can't see anything in drive except what we created.) It used to be "full permissions to your entire drive"! 😱

All this said, did you have an actual problem with the permissions as written? Did it fail in a specific way?

**Comment by lorennorman** [comment]
Yes, CI is failing, we've let it drift a bit with some big recent changes. Don't worry about it, unless you're in the mood to worry about it!

**Review by lorennorman — CHANGES_REQUESTED** [review]
I don't want to modify the Google permissions bit until we're a little clearer on what's needed/desired, but the rest is great.

File: README.md
Comment (nathanleiby) [reply README.md:21]: Here's a screenshot of the "test user" UX, for reference:
OAuth_consent_screen_–_APIs___Services_–_paperize-dev_–_Google_API_Console

File: README.md
Comment (nathanleiby) [reply README.md:34]: I ran into an issue with `Google Sheets API ../auth/drive.file`, specifically. I wasn't able to find a permissions for "Google Sheets API" that had this scope name, (`../auth/drive.file`). I suspect it was a mistakenly copied from the scope above. I *think* scope names a globally unique.

Update_selected_scopes_–_APIs___Services_–_paperize-dev_–_Google_API_Console

--

If you have a chance to look at how the production app's OAuth setup is configured, I'm happy to just mirror those scopes in the "getting started" guide.

Comment (lorennorman) [reply README.md:34]: I left this hanging! I'm coming through to get it right now, here's the production scopes:
image

Not a single sensitive scope, no verification required:
Screen Shot 2022-02-26 at 10 16 21 AM

Going to update the README to only `/auth/drive.file`

**Pull request merged**
